src/components/formik/inputs/LicenseDropdownControl.jsx

```diff
@@ -27,7 +27,7 @@ export const LicenseDropdownControl = (props) => {
 
   return (
     <span style={remainingStyle()}>
-      {`${title}   (${used} remaining out of ${total})`}
+      {`${title}   (${used} assigned of ${total})`}
```

Comment:
Will's comment in the ticket says 
> The difference between the Quantity and the sum of the assigned licenses should be displayed in the list of Titles that have not yet been added to a Licensee.

So I think they're looking for something like 
```javascript
{`${title} (${total - used} remaining)`}
```

Reply:
I think he wanted to use the work assigned, the ticket has an extra comment about that.